File: WeLearnApp/src/main/java/com/welearn/WeLearnApp/dto/response/UserProfileResponse.java
package com.welearn.WeLearnApp.dto.response;

import lombok.*;
import lombok.experimental.FieldDefaults;

import java.time.LocalDate;

@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
@FieldDefaults(level = AccessLevel.PRIVATE)
public class UserProfileResponse {
    String id;
    String firstName;
    String lastName;
    LocalDate dob;
    String phoneNumber;
    LocationResponse location;
    String email;
    String avatarUrl;
}

File: WeLearnApp/desktop-app/TutorApp/Models/ForAPI/Response/UserProfileResponse.cs
using Microsoft.VisualBasic;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using TutorApp.Models.ForAPI.Response;

namespace TutorApp.Models.ForAPI.Response
{
    public class UserProfileResponse
    {
        public string? firstName { get; set; }
        public string? lastName { get; set; }
        public DateTime? dob { get; set; }
        public string? phoneNumber { get; set; }
        public LocationResponse? location { get; set; }
    }
}
